### PR TITLE
Added issue template

### PR DESCRIPTION
**Pull request by digeff**
According to this: https://github.com/blog/2111-issue-and-pull-request-templates
This should be a template for when someone creates a new issue.

I don't know how to test this before pushing this though :/

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+- React-Native tools version: 
+- VSCode version: 
+- OS version:
```
Comment (hamiltonia) [inline .github/ISSUE_TEMPLATE.md]: OS platform:

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+- React-Native tools version: 
+- VSCode version: 
+- OS version: 
+- React Native version: 
+- Output of the Debug Console (View -> Toogle Debug Console):
```
Comment (hamiltonia) [inline .github/ISSUE_TEMPLATE.md]: Probably add (optional) to the output items, as they won't always be necessary.

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+- React-Native tools version: 
+- VSCode version: 
+- OS version: 
+- React Native version: 
+- Output of the Debug Console (View -> Toogle Debug Console): 
+- Output of the React-Native output channel (View -> Toogle Output -> Select React-Native in ListBox): 
+- Output of the Developer Tools console (Help -> Toggle Developer Tools -> Select Console tab): 
+- Does any other feature seem to not be working as expected?
```
Comment (hamiltonia) [inline .github/ISSUE_TEMPLATE.md]: Why is this needed? If they have other issues, either create a new issue, or describe below.

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (digeff) [inline .github/ISSUE_TEMPLATE.md]: You mean as another line, or rename line 3 to that?

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (digeff) [inline .github/ISSUE_TEMPLATE.md]: What do you think of adding (If relevant) instead?
I don't want someone to think: It's optional, so I don't need to send it...

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (digeff) [inline .github/ISSUE_TEMPLATE.md]: Changing to OS platform and version until I understand what you meant here.

File: .github/ISSUE_TEMPLATE.md (hunk 3 above)
Comment (digeff) [inline .github/ISSUE_TEMPLATE.md]: Removed!

File: .github/ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+- React-Native tools version:
```
Comment (meenakb) [inline .github/ISSUE_TEMPLATE.md]: Leave it at "React-Native version". We are interested in react-native's version and NOT react-native-cli's version?

Comment (digeff) [inline .github/ISSUE_TEMPLATE.md]: Actually this line is meant to be our extension.
I don't think we usually care about the react-native-cli version, it usually just passes stuff through to the react-native library.

Comment (meenakb) [inline .github/ISSUE_TEMPLATE.md]: Then, make the title less ambiguous. "React-native extension version"? Also, wouldn't react-native version be a necessary data point?

Comment (digeff) [inline .github/ISSUE_TEMPLATE.md]: Yes, that's on line 4.

**Comment by meenakb** [comment]
:+1:

**Comment by hamiltonia** [comment]
:+1:
